spark/src/main/scala/geotrellis/spark/TileLayerMetadata.scala

```diff
@@ -165,6 +165,21 @@ object TileLayerMetadata {
   }
 
   def fromRdd[
+    K: (? => TilerKeyMethods[K, K2]) ,
+    V <: CellGrid,
+    K2: SpatialComponent: Boundable
+  ](rdd: RDD[(K, V)], crs: CRS, scheme: ZoomedLayoutScheme, maxZoom: Option[Int] = None):
```

Comment:
@lossyrob sorry for the delay, I just updated the branch with your suggested changes.